package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@angular-devkit/schematics": "^12.2.2",
     "@typescript-eslint/experimental-utils": "^4.31.1",
-    "eslint-etc": "^4.1.0",
+    "eslint-etc": "^4.2.4",
```

Comment:
Even without this updating, when I ran `npm i`, all the changes in `package-lock.json` made on #212, are reverted as you can see above. Thoughts?